views/includes/entity.html

```diff
@@ -62,8 +67,9 @@ <h2 class="govuk-heading-l">Overseas entity details</h2>
           html: formattedPrincipalAddressHtml
         },
         actions: {
-          items: [CREATE_CHANGE_LINK(
-            OE_CONFIGS.ENTITY_CHANGE_PRINCIPAL_ADDRESS,
+          items: [CREATE_PAGE_CHANGE_LINK(
+            changeLinkUrl,
+            OE_CONFIGS.CHANGE_PRINCIPAL_ADDRESS,
```

Comment:
the utils/page.change.link.ts looks very similar to the existing utils/change.link.ts

you could just concatenate changeLinkUrl + OE_CONFIGS.CHANGE_PRINCIPAL_ADDRESS and use the existing CREATE_CHANGE_LINK

eg.
`items: [CREATE_CHANGE_LINK(
            changeLinkUrl + OE_CONFIGS.INCORPORATION_COUNTRY,`

Reply:
We're using it to have change links for register-an-overseas-entity and update-an-overseas-entity paths but the existing change links does the concat.

Reply:
Previously we were passing the full url (register-an-overseas-entity/entity#incorporation_country)(OE_CONFIGS.ENTITY_CHANGE_COUNTRY) and we did not want to expose the concatenation
logic in the view , but this should also work and with new changes for summaryList also added to utils/change.link.ts would make more sense to have all the changes in one place...